.github/workflows/rust.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - name: Check Cargo.lock for datafusion-cli
         run: |
           # If this test fails, try running `cargo update` in the `datafusion-cli` directory
-          cargo check --manifest-path datafusion-cli/Cargo.toml --locked
```

Comment:
Without this change CI tests fail. If you have any other suggestions to make CI tests work. I can change it accordingly.